yt/yt/server/controller_agent/controllers/operation_controller_detail.cpp

```diff
@@ -10057,6 +10057,20 @@ double TOperationControllerBase::GetCpuLimit(const TUserJobSpecPtr& userJobSpec)
     return std::max(userJobSpec->CpuLimit, Options_->MinCpuLimit);
 }
 
+std::pair<std::optional<TString>, bool> TOperationControllerBase::NormalizeDockerImage(
```

Comment:
Let's introduce a struct for this pair, since return value is unclear if you see method name only.